opa/opa.go

```diff
@@ -78,7 +93,26 @@ func (a *restAuthorizer) Authorize(subject string, groups []string, permission r
 		return http.StatusForbidden, false
 	}
 
-	res, err := a.client.Post(a.url.String(), "application/json", bytes.NewBuffer(j))
+	req, err := http.NewRequest(http.MethodPost, a.url.String(), bytes.NewBuffer(j))
+	if err != nil {
+		level.Error(a.logger).Log("msg", "failed to build authorization request", "err", err.Error())
+
+		return http.StatusForbidden, false
```

Comment:
This should be an internal server error, not a forbidden error. This is important because forbidden is a 4xx error, meaning clients won't retry requests, but this is clearly a transient error.